### DISCUSSION TITLE
[BUG] Docker Unable to find proxy file

### DISCUSSION THREAD

**Discussion by 123bistami** [mubeng/mubeng#85] [Q&A]
i installed the package via docker pull.

Then run command `sudo docker run kitabisa/mubeng [-c|-a :8080] -f file.txt Error! no proxy file provided. ` the file is in the home directory tried ./file.txt and /home/ubuntu/file.txt same error. I think ignore -f flag are ignored.

then i installed via cloning the reprosotory
Everything works but i can run it as daemon get error: exit status 1

Pls help and thank you for sharing your work.

One last question, is there a way to log the response of request e.g. to log the requested address and response code 200,403,500

**Comment by dwisiswant0** [comment]
1. You should mount your local directory first. Try with `docker run --rm -it -v ${HOME}:/ kitabisa/mubeng [-c|-a :8080] -f file.txt`, and let us know.

2. Does the daemon run inside a Docker container?

3. No, verbose (`-v/--verbose`) does not log I/O requests/responses from the proxy. See [NOTES](https://github.com/kitabisa/mubeng#notes).

**Reply by 123bistami** [reply]
After 1. i get this error

docker: Error response from daemon: invalid volume specification: '/home/ubuntu:/': invalid mount config for type "bind": invalid specification: destination can't be '/'.
See 'docker run --help'.
-a: command not found

2. i installed via command docker pull package_name, if i run docker images kitabisa/mubeng image is listed. I am not a docker Expert, do i have todo something else after pull package.
I installed docker via `sudo apt install docker.io`
Thank you

**Reply by 123bistami** [reply]
Any idea?

**Comment by j4i0n4** [comment]
me too
Error! no proxy file provided.
